src/index.js

```diff
@@ -1478,6 +1487,24 @@ const initializeFormElements = () => {
    *  PPOM
    */
 
+  // Mint ERC20 in Sepolia
+  mintERC20.onclick = async () => {
```

Comment:
should the `mintERC20` be renamed to provide more context? Currently the button only displays if we are in Sepolia, and the erc20 contract is set to a the [Sepolia one](https://github.com/MetaMask/test-dapp/pull/298/files#diff-972434016582e4f749542bddfc5a1473440bbdcdd8f32ddfdb5f3bfd239a2f4bR17). So `mintSepoliaERC20` seems to be more appropriate.
What do you think?